src/main/java/mate/academy/model/User.java

```diff
@@ -0,0 +1,71 @@
+package mate.academy.model;
+
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.Table;
+
+@Entity
+@Table(name = "users")
+public class User {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+    private String name;
+    private String password;
+    private String email;
```

Comment:
```suggestion
    @Column(nullable = false)
    private String password;
    @Column(nullable = false, unique = true)
    private String email;
```
Let’s  add some constraints